pdsfile/pds3file/rules/JNOSRU_xxxx.py

```
@@ -0,0 +1,105 @@
+            r'volumes/JNOSRU_xxxx\1/\2/SRU_EDR/\3'),
+    # image files to count rate files
+    (r'volumes/JNOSRU_xxxx(|_v[\d\.]+)/(JNOSRU_0\d\d\d/DATA)/SRU_EDR/(.*)/\w+\.(FIT|LBL)', 0,
+            r'volumes/JNOSRU_xxxx\1/\2/SRU_COUNTRATE_TABLE/\3/*.CSV'),
+    (r'volumes/JNOSRU_xxxx(|_v[\d\.]+)/(JNOSRU_0\d\d\d/DATA)/SRU_EDR/([^\.]*)', 0,
+            r'volumes/JNOSRU_xxxx\1/\2/SRU_COUNTRATE_TABLE/\3/*.CSV'),
+])
+
+associations_to_previews = translator.TranslatorByRegex([
+    # image files to previews
+    (r'volumes/JNOSRU_xxxx(|_v[\d\.]+)/(JNOSRU_0\d\d\d/DATA/SRU_EDR/\w+/\w+/SRU_\d+_20\d{5}T\d{6}_\d\d_V\d\d)\.(FIT|LBL)', 0,
+            [r'previews/\2_full.jpg',
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)